src/custom/state/orders/hooks.ts

```diff
@@ -95,3 +114,20 @@ export const useClearOrders = (): ClearOrdersCallback => {
   const dispatch = useDispatch<AppDispatch>()
   return useCallback((clearOrdersParams: ClearOrdersParams) => dispatch(clearOrders(clearOrdersParams)), [dispatch])
 }
+
+export const useLastCheckedBlock = ({ chainId }: GetLastCheckedBlockParams): number => {
+  return useSelector<AppState, number>(state => {
+    if (!chainId) return 0
+
+    return state.orders?.[chainId]?.lastCheckedBlock ?? ContractDeploymentBlocks[chainId] ?? 0
+  })
+}
+
+export const useUpdateLastCheckedBlock = (): UpdateLastCheckedBlockCallback => {
+  const dispatch = useDispatch<AppDispatch>()
+  return useCallback(
+    (updateLastCheckedBlockParams: UpdateLastCheckedBlockParams) =>
+      dispatch(updateLastCheckedBlock(updateLastCheckedBlockParams)),
+    [dispatch]
```

Comment:
is dispatch not stable?

Reply:
It is, but eslint still wants it in deps